.github/workflows/build_and_deploy.yml

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2022 ETH Zurich
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+name: Build, test and deploy docker images
+
+# The builds are triggered with pushes on the 'main' branch or with a pull
+# request against the 'main' branch
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+env:
+  base_version: 1
+  base_image_name: pika-ci-base
+  hip_version: 1
```

Comment:
Is there a use case for different versions for the base image and the HIP image? Since the HIP image is rebuilt based on the base image I would expect we always need to bump the hip version as well.
```suggestion
```

Reply:
I was thinking that if we just change rocm, then it avoid to always bump the main one, I don't have a strong opinion on that

Reply:
That's true, I didn't think about that. However, I think to keep things simple it's enough to have just one version number.